packages/shared/src/styles/constants.scss

```diff
@@ -41,6 +42,7 @@ $color-green-1: #4bb4b3;
 $color-green-2: #3d9494;
 $color-green-3: #00a79e;
 $color-green-4: #008079;
+$color-green-5: #4bb4b329;
```

Comment:
Same question here

Reply:
It translates to  RGBA color - rgba(75, 180, 179, 0.16)

_Image from figma:_
![Screen Shot 2022-04-07 at 9 24 05 AM](https://user-images.githubusercontent.com/98398322/162126145-a2776048-81dd-4f72-ae36-4a3d6287eeb3.png)
)

Reply:
Where did you get this value? Please get it from Figma, it should have a 6 digit hexadecimal value.

Reply:
Same here. You don't need to add a color for opacity **only** since #4bb4b3 already exists as $color-green-1.

![image](https://user-images.githubusercontent.com/58209918/162371422-d0a1fd4c-e881-440f-b6aa-a50957699ab5.png)

Reply:
You don't need to add a new color for this is what I'm saying, you can use the existing color and add opacity.

Reply:
ok. will work on that